test/tests/userobjects/particle_stepper/boris_stepper/tests

```diff
@@ -31,14 +55,38 @@
     [particle]
       type = 'CSVDiff'
       input = 'cyclotron_motion.i'
-      csvdiff = 'cyclotron_motion_csv_particle_data_0011.csv'
+      csvdiff = 'cyclotron_motion_csv_particle_data.csv'
       should_execute = False
       prereq = 'cyclotron_motion/fields'
       detail = 'and verify particle motion'
     []
   []
   [circular_e_field]
     requirement = 'The system shall be capable of accurately capturing the path of charged particles in a time varrying electric field'
+    [convergence]
+      type = PythonUnitTest
+      input = 'test.py'
+      heavy = True
+      test_case = BorisCyclotronElectricTest
+      required_python_packages = 'pandas matplotlib'
+      installation_type = in_tree
+    []
+    [convergence_errors]
+      type = 'CSVDiff'
+      input = 'test.py'
+      heavy = True
+      csvdiff = 'circular_e_field_errors.csv'
+      should_execute = False
+      prereq = 'circular_e_field/convergence'
+    []
+    [convergence_orders]
+      type = 'CSVDiff'
+      input = 'test.py'
+      heavy = True
+      csvdiff = 'circular_e_field_orders.csv'
+      should_execute = False
+      prereq = 'circular_e_field/convergence'
+    []
```

Comment:
Same comments as with the other convergence tests above about `heavy = true` and the need for `detail`.